bcipy/helpers/report.py

```diff
@@ -0,0 +1,220 @@
+import io
+from abc import ABC
+from typing import List, Optional
+
+from matplotlib.figure import Figure
+from reportlab.platypus import SimpleDocTemplate, Paragraph, Image
+from reportlab.lib.pagesizes import letter
+from reportlab.lib.styles import getSampleStyleSheet
+from reportlab.platypus import Flowable, KeepTogether
+from reportlab.lib.units import inch
+
+from bcipy.config import BCIPY_FULL_LOGO_PATH
+
+
+class ReportSection(ABC):
+    """Report Section.
+
+    An abstract class to handle the creation of a section in a BciPy Report.
+    """
+
+    def _compile(self) -> Flowable:
+        ...
+
+    def _create_header(self) -> Flowable:
+        ...
```

Comment:
Given that the `_compile` and `_create_header` methods are intended to be called by an external class (`Report`), should this still have a leading underscore?

Reply:
The intention is that it should not be called individually, but only on a final build. The create header is something that should only be called internally. I will remove the underscore to compile and add a note in the docstring.